presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -146,17 +150,28 @@ class EventViewModel @Inject constructor(
             remindAt = remindAt.toEpochSecond(),
             eventCreatorId = preferenceRepository.retrieveCurrentUserOrNull()?.userId ?: "",
             isUserEventCreator = false,
+            isGoing = true,
             attendees = listOf( /** TODO Mock data until we have added real attendees */
                 Attendee(1, "email", "job blogs", UUID.randomUUID().toString(), UUID.randomUUID().toString(), true, 4L),
                 Attendee(2, "gmail", "peter rab", UUID.randomUUID().toString(), UUID.randomUUID().toString(), false, 2L)),
             photos = eventScreenState.value.listOfPhotoUri
         )
 
         viewModelScope.launch {
-            eventRepository.insertEvent(event)
-
-            val alarmItem = event.toAlarmItem(AgendaType.EVENT)
-            alarmScheduler.scheduleAlarmReminder(alarmItem)
+            when(val responseState = eventRepository.insertEvent(event)) {
```

Comment:
In what cases will this ever return something different than success?
Since you are just inserting to a db

Reply:
@MKiperszmid I can remove it if there is nothing that can go wrong with an insert. @philipplackner

Reply:
The only thing I can think of is if the device is out of storage